### PR TITLE
#2329 Create db connection to Alembic for the CC team

### PR DESCRIPTION
**Pull request by agile-josiah**
## What was the problem?
Draft pr for dev: CC team needs to connect to the rds db for use with alembic to create tables and 

Associated tickets or Slack threads:
- #2329

## How does this fix it?[^1]
Adds refs to the secrets in lens, currently only in dev

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

**Review by tejans24 — APPROVED** [review]

File: helm/_shared/named_templates/_postgres.tpl [hunk 1]
```diff
@@ -60,3 +60,20 @@
...
+    secretKeyRef:
+      name: domain-cc-db
+      key: DOMAIN_CC_USER
+- name: DOMAIN_CC_PW
+  valueFrom:
+    secretKeyRef:
+      name: domain-cc-db
+      key: DOMAIN_CC_PW
+- name: ALEMBIC_SCHEMA
+  value: {{ .Values.global.service.db.domainCcSchemaName }}
```
Comment (msnwatson) [inline helm/_shared/named_templates/_postgres.tpl]: Nit: new line at EOF

**Review by msnwatson — CHANGES_REQUESTED** [review]
It's a little confusing but the named templates are symlinked across all the deployments I think so we'll want to make a new file just in the CC directory

File: helm/_shared/named_templates/_postgres.tpl (hunk 1 above)
Comment (agile-josiah) [reply helm/_shared/named_templates/_postgres.tpl]: Done

**Review by msnwatson — APPROVED** [review]

File: helm/domain-cc/templates/deployment.yaml [hunk 2]
```diff
@@ -29,6 +29,8 @@ spec:
             {{- include "vro.commonEnvVars" . | nindent 12 }}
             {{- include "vro.mqClient.envVars" . | nindent 12 }}
             {{- include "vro.redisClient.envVars" . | nindent 12 }}
+            {{- include "domainCc.alembic.envVars" . | nindent 12 }}
+            {{- include "vro.postgresUrl" . | nindent 12 }}
```
Comment (lukey-luke) [inline helm/domain-cc/templates/deployment.yaml:33]: FYSA @tyler-spangler6 this allows us to make use of `DB_URL` from env vars (e.g. `os.environ.get()` in python)
See linked file [here](https://github.com/department-of-veterans-affairs/abd-vro/blob/032c4eca676bc22121573cba7033749c815b0c08/helm/_shared/named_templates/_postgres.tpl#L5).

**Review by lukey-luke — APPROVED** [review]

**Pull request merged**